docs/book/src/plugins/declarative-v1.md

```diff
@@ -0,0 +1,64 @@
+# Declarative Plugin
```

Comment:
Shows to be missing be rebased with master and squash all commits
See that this one should not be part of this PR.

Reply:
I will fix this asap